inyoka_theme_default/templates/portal/privmsg/index.html

```diff
@@ -0,0 +1,209 @@
+{#
+  portal/privmsg/index.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Display a list of private messages.
+
+  :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+
+{% import 'macros.html' as macros %}
+
+{% extends 'portal/privmsg/base.html' %}
+
+{% block breadcrumb %}
+  {{ macros.breadcrumb_item(_('Portal'), href('portal', 'privmsg', 'inbox')) }}
+{% endblock %}
+
+{% block content %}
+  {%- if message %}
+    <div class="message">
+      <div class="head">
+        <div class="row">
+          <div class="col-sm-9">
+            <dl>
+              <dt>{% trans %}From:{% endtrans %}</dt>
+              <dd>
+                <a{% if not message.author.is_active %} class="user_inactive"{% endif %} href="{{ message.author|url }}">
+                  {{- message.author|e }}
```

Comment:
`-` can be remove from `{{-` or `-%}` etc. (the `-` just removes white space before/after. However, we readability over nice rendered html). sed will be your friend imho. ;)